variables.tf

```diff
@@ -196,23 +200,23 @@ variable "dcos_base_download_url" {
 
 variable "bootstrap_id" {
   description = "bootstrap id that is used to download the bootstrap files"
```

Comment:
Hi, how do you get this bootstrap id?
I cant seem to find any link in official documentaion for dcos about it